app/components/filters/include_new.rb

```diff
@@ -0,0 +1,13 @@
+module Filters
+  class IncludeNew
+    def call
+      ETL::Filter.new(filter_name: 'Existing record') do |record|
+        new(record)
+      end
+    end
+
+    def new(record)
+      !Facts::Booking.find_by(reference_number: record['id'])
```

Comment:
If we just want to know if a record exists we can use the `Facts::Booking.exists?`. We don't need to return the object.